subsys/bluetooth/mesh/friend.c

```diff
@@ -1057,6 +1078,42 @@ int bt_mesh_friend_init(void)
 	return 0;
 }
 
+static bool is_segack(struct net_buf *buf, u64_t *seqauth, u16_t src)
+{
+	if (buf->len != 16) {
+		return false;
+	}
+
+	bool found = false;
+
+	struct net_buf_simple_state state;
```

Comment:
Both `found` and `state` declarations to the beginning of the function, preferably in the "inverted x-mas tree" style, i.e. `state` first.